File: 135022193-main/2023/CS50 Done/mario-more/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int n;
    //prompt user for size between 0 and 9
    do
    {
        n = get_int("size: ");
    }
    while (n < 1 || n > 8);

    //loop for creating the rows
    for (int i = 0; i < n; i++)
    {
        //loop for creating the first blank spaces
        for (int j = n - 1; j > i; j--)
        {
            printf(" ");
        }
        //loop for creating the first pyramid of hashes
        for (int p = 0; p <= i; p++)
        {
            printf("#");
        }
        printf("  ");
        //loop for creating the second pyramid of hashes
        for (int e = 0; e <= i; e++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: 135022193-main/2024/Done/sentimental-mario-less/mario.py
import cs50


def main():
    n = get_height()
    for i in range(n):
        print(" " * (n - (i + 1)) + "#" * (i + 1))


def get_height():
    while True:
        n = cs50.get_int("Height: ")
        if n > 0 and n < 9:
            return n


main()
